include/rbs/constants.h

```diff
@@ -0,0 +1,82 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
I'm surprised we're generating these files in the same PR, won't we have conflict when building?

Same for all what's under `include/` and `src/`